Cargo.lock

```diff
@@ -1754,6 +1757,15 @@ dependencies = [
  "windows-sys 0.52.0",
 ]
 
+[[package]]
+name = "is_executable"
```

Comment:
This is owned by Nick Fitzgerald, so trustworthy IMO.
https://github.com/fitzgen/is_executable